src/vs/workbench/contrib/welcomeGettingStarted/common/gettingStartedContent.ts

```diff
@@ -385,6 +385,14 @@ export const walkthroughs: GettingStartedWalkthroughContent = [
 						type: 'svg', altText: 'VS Code Settings', path: 'settings.svg'
 					},
 				},
+				{
+					id: 'profiles',
+					title: localize('gettingStarted.profiles.title', "Customize your workspace with Profiles"),
+					description: localize('gettingStarted.profiles.description.interpolated', "Profiles let you customize the numerous settings and extensions in VS Code and quickly switch between them. Create your own profile from scratch or use the predefined set of templates for your specific workflow.\n{0}", Button(localize('tryProfiles', "Try Profiles"), 'command:workbench.profiles.actions.createProfile')),
```

Comment:
Sounds good. Here's a slight re-word suggestion that might help with clarity:
"Profiles let you create sets of VS Code customizations that include settings, extensions and UI state. Create your own profile from scratch or use the predefined set of profile templates for your specific workflow."

Feel free to edit it though.